backend/de.metas.edi/src/main/sql/postgresql/system/80-de.metas.edi/5749050_indices_for_preformance_on_shipment_completion.sql

```diff
@@ -0,0 +1,28 @@
+CREATE INDEX IF NOT EXISTS EDI_Desadv_Pack_M_HU_ID on edi_desadv_pack (m_hu_id)
+;
+
+CREATE INDEX IF NOT EXISTS M_Package_HU_HU_ID on M_Package_HU (m_hu_id)
+;
+
+CREATE INDEX IF NOT EXISTS m_hu_m_hu_pi_version_id ON m_hu(m_hu_pi_version_id)
```

Comment:
not sure about this index; for each `m_hu_pi_version_id` there will be a large amount of HUs, wo the DB will habe to scan a large part of that index

Reply:
as talked, we will keep it